OpenSim/Common/DataTable.h

```diff
@@ -1350,26 +1355,42 @@ class DataTable_ : public AbstractDataTable {
 
     // Split element into constituent components and append the components to
     // the given vector. For example Vec3 has 3 components.
-    template<int N>
+    template<int N, typename Iter>
     static
-    void splitElementAndPushBack(std::vector<double>& row,
-                                 const SimTK::Vec<N>& elem) {
-        for(unsigned i = 0; i < N; ++i)
-            row.push_back(elem[i]);
+    void splitElementAndAppend(Iter begin, Iter end, 
```

Comment:
I wouldn't say this function is appending. What about `splitElementAndCopy()`?

Reply:
I like that better, good suggestion.